components/register/all.go

```diff
@@ -8,6 +8,7 @@ import (
 	_ "go.viam.com/rdk/components/encoder/register"
 	_ "go.viam.com/rdk/components/gantry/register"
 	_ "go.viam.com/rdk/components/generic/register"
+	_ "go.viam.com/rdk/components/gripper/register"
```

Comment:
Do you know why this changed? There is no change to the gripper code in this PR

Reply:
I moved it from register/all_cgo.go

(I think we needed it for building a sample)

the original `no_cgo` port painted with a broad brush, and I've been selectively adding things back as needed